pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -0,0 +1,552 @@
+version 1.0
+
+import "../../../pipelines/broad/rna_seq/UMIAwareDuplicateMarking.wdl" as UmiMD
+import "../../../tasks/broad/Utilities.wdl" as utils
+
+## Copyright Broad Institute, 2021
+##
+## This WDL pipeline implements data processing for RNA with UMIs
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation.
+## For program versions, see docker containers.
+##
+## LICENSING :
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+workflow RNAWithUMIsPipeline {
+
+	String pipeline_version = "0.1.0"
+
+	input {
+		File? bam
+		File? r1_fastq
+		File? r2_fastq
+		String read1Structure
```

Comment:
Can we add a parameter_meta { section so that the inputs are defined in the workflow. For example, it's not obvious looking at the code what the read1Structure string should be.

Reply:
Created a Jira ticket: https://broadinstitute.atlassian.net/browse/GL-1879 to track this.